similarity/process_text.py

```diff
@@ -0,0 +1,12 @@
+def process_text(name):
+  with open(name + '.txt', 'r') as input:
+    lines = input.readlines()
+  content = [l.strip() for l in lines]
+  comma_removed = [c.replace(',', '') for c in content]
+  replace_colon = [s.replace(':', ',') for s in comma_removed]
+
+  with open(name + '.csv', 'w') as output:
+    for i in replace_colon:
```

Comment:
Consider performing the preprocessing lazily instead of materializing the intermediate lists, e.g.

```python
for i in map(lambda line: line.strip().replace(',', ' ').replace(':', ','), lines)
```